libs/wingc/src/closure_transform.rs

```diff
@@ -195,6 +195,33 @@ impl Fold for ClosureTransformer {
 				// we need to set this to false.
 				new_func_def.is_static = false;
 
+				let class_init_body = vec![Stmt {
```

Comment:
Comment of what we're doing here `this.display.hidden = true`?
Also kind of feels like we can benefit from calling tree-sitter+parser to generate this ast from a string. Probably felt weird coding like this...

Reply:
It did feel quite weird, yeah :)

Hmm, it didn't occur to me that maybe we could generate this AST from a string directly inside this transform... let me take a look into this

Reply:
Added a comment.

I'll probably leave the other suggested changes out of this PR because I think it'll require refactoring code in the parser so that AST nodes are created with the correct spans. The other caveat is even if we could generate an AST from a string, it's not clear doing this in the closure transform makes sense considering it would require the compiler to do more work (lexing and parsing a string) when compared to creating a set of structs directly.